client/src/Component/SMSModal.js

```diff
@@ -1,23 +1,54 @@
-/* eslint-disable jsx-a11y/label-has-for */
+
 import { useState } from "react";
 import Modal from "react-bootstrap/Modal";
 import Button from "react-bootstrap/Button";
 
+
+
+
 function SMSModal(props) {
     const [show, setShow] = useState(false);
     const hideModal = () => setShow(false);
     const showModal = () => setShow(true);
```

Comment:
Let's move these functions after the `useState` and `useEffect` functions, which typically come first in React components